_test/test/goldens/generated_build_script.dart

```diff
@@ -109,13 +109,14 @@ final _builders = <_i1.BuilderApplication>[
         r'test/**.node_test.dart',
         r'test/**.vm_test.dart'
       ]),
-      defaultOptions: _i8.BuilderOptions({
-        'dart2js_args': ['--minify']
+      defaultOptions: _i8.BuilderOptions(const {
```

Comment:
Note that `BuilderOptions` has a const constructor - I would probably try making the builder options themselves be const if we want to move towards more const things here.